src/helpers/reports.js

```diff
@@ -57,9 +57,41 @@ export const getSupportLevel = (percentile) => {
       support_level = 'Needs Extra Support';
       tag_color = supportLevelColors.below;
     }
+  } else if (rawScore !== undefined && grade >= 6) {
+    const { above, some } = getRawScoreThreshold(taskId);
+    if (rawScore >= above) {
+      support_level = 'At or Above Average';
+      tag_color = supportLevelColors.above;
+    } else if (rawScore > some && percentile < above) {
```

Comment:
I think that both of these variables should be `rawScore`, correct?
```suggestion
    } else if (rawScore > some && rawScore < above) {
```

Reply:
I think this one is still unresolved.